starboard/CHANGELOG.md

```diff
@@ -66,6 +66,10 @@ This API is unused by Cobalt and has been deprecated.
 By supporting the keycode for the remote record button on Cobalt, the webapp can be
 programmed to respond to the remote record button in the future.
 
+### Rmoved SbSystemDeviceType and add kSbSystemPropertyDeviceType
+This enumeration of device types SbSystemDeviceType has been deprecated. We
+add new system property for device type as a string.
```

Comment:
s/add/added/ and please mention the newly defined property name `kSbSystemPropertyDeviceType`. Perhaps worth mentioning that the allowed values are now checked by nplb tests ?

Reply:
Oh also i see linter warned about typo in Rmoved => Removed